### PR TITLE
Obtaining Image From Gallery

### PR DESCRIPTION
**Pull request by TedHoryczun**
Getting an image from the gallery. Then presenting it on screen as a focused image.

### CODE REVIEW

File: app/src/main/java/hypr/a255bits/com/hypr/MainInteractor.kt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package hypr.a255bits.com.hypr
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.net.Uri
+import android.provider.MediaStore
+
+/**
+ * Created by ted on 7/17/17.
```
Comment (martyn) [inline app/src/main/java/hypr/a255bits/com/hypr/MainInteractor.kt]: lets remove attribution comments(we have git)

File: app/src/main/java/hypr/a255bits/com/hypr/MainMvp.kt [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package hypr.a255bits.com.hypr
+
+import android.graphics.Bitmap
+import android.net.Uri
+
+/**
+ * Created by ted on 7/17/17.
```
Comment (martyn) [inline app/src/main/java/hypr/a255bits/com/hypr/MainMvp.kt]: remove

File: app/src/main/java/hypr/a255bits/com/hypr/MainPresenter.kt [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package hypr.a255bits.com.hypr
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.net.Uri
+
+/**
+ * Created by ted on 7/17/17.
```
Comment (martyn) [inline app/src/main/java/hypr/a255bits/com/hypr/MainPresenter.kt]: remove

File: app/src/main/java/hypr/a255bits/com/hypr/MainActivity.kt [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+    }
+
+
+    override fun displayGallery() {
+        val intent = Intent(Intent.ACTION_PICK, galleryFileLocation)
+        startActivityForResult(intent, RESULT_GET_IMAGE)
+    }
+
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
```
Comment (martyn) [inline app/src/main/java/hypr/a255bits/com/hypr/MainActivity.kt]: s/data/intent (method variable name)

**Review by martyn — CHANGES_REQUESTED** [review]
Some small issues but the code looks good!

File: app/src/main/java/hypr/a255bits/com/hypr/MainActivity.kt (hunk 4 above)
Comment (mikkel) [reply app/src/main/java/hypr/a255bits/com/hypr/MainActivity.kt]: > s/data/intent = rename `data` to `intent`

**Comment by mikkel** [comment]
LGTM :+1:

screenshot from 2017-07-17 17-11-33

**Review by martyn — APPROVED** [review]

**Pull request merged**